cmake/modules/hunter_download.cmake

```diff
@@ -79,6 +81,15 @@ function(hunter_download)
   set(HUNTER_PACKAGE_VERSION "${HUNTER_${h_name}_VERSION}")
   set(ver "${HUNTER_PACKAGE_VERSION}")
   set(HUNTER_PACKAGE_URL "${HUNTER_${h_name}_URL}")
+  if(use_download_server)
```

Comment:
You can check `if(use_download_server)` inside `hunter_download_server_url` funtion.